packages/transition-frontend/src/components/map/TransitionMainMap.tsx

```diff
@@ -70,8 +70,13 @@ class MainMap extends React.Component<MainMapProps & WithTranslation & PropsWith
     constructor(props: MainMapProps & WithTranslation) {
         super(props);
 
+        // Get layers for section from config first, fall back to user preferences for backward compatibility
```

Comment:
Here we don't need backward compatiobility because as soon as this PR is merged, preferences will never be read

Reply:
removed